api/build.gradle

```diff
@@ -0,0 +1,115 @@
+
+plugins {
+    id 'antlr'
+    id 'checkstyle'
+    id 'org.springframework.boot' version '3.4.1'
+    id 'com.gorylenko.gradle-git-properties' version "2.4.2"
+}
+
+dependencies {
+    implementation project(":contract")
+    implementation project(":frontend")
```

Comment:
api shouldn't depend on the frontend

Reply:
On prod build should

Reply:
hmmm not quite, in maven there's a `skipUIBuild` flag to skip UI build even in prod profile. Let's address this functionality

Reply:
not sure why'd we need skipUIBuild having a local profile, resolving this